### DISCUSSION TITLE
Pdbs Prometheus-Operator

### DISCUSSION THREAD

**Discussion by sag-a** [prometheus-operator/prometheus-operator#5764] [General]
Hello, I would like to ask if it is possible to use the prometheus-operator and create through him pod distruptions budgets for the deployments which are operated by him or which is the proposed way to do it. Thank you.

**Comment by sag-a** [comment]
I have replicas:2 but I don't see any pdb in my cluster

**Comment by simonpasquier** [comment]
you can follow https://github.com/prometheus-operator/prometheus-operator/issues/3917 which already tracks the feature request.

**Reply by sag-a** [reply]
I see thanks a lot for the reply.
